themes/OpenFisca-2022/layouts/partials/card.html

```diff
@@ -8,13 +8,17 @@
 {{ with .author }}{{ with .type }}{{ $authorType = . }}{{ end }}{{ end }}
 {{ $authorTypeIcon := index .Site.Params.showcase.author.types.icons $authorType }}
 
+{{ $package := index $.Site.Data.packages .package }}
+
 <div class="card {{ if .website }}{{ if not .until }}card-hasLink{{ end }}{{ end }}">
 	<div class="card_image">
 		<img class="screenshot" src="{{ $imageSrc }}" alt="{{ .title }}" />
 	</div>
 	<div class="card_titleWrapper">
 		<h3 class="card_title">{{ .title }}</h3>
-		<div class="card_flag flag flag-{{ .country }}"><span>{{ .country }}</span></div>
+		{{ with $package }}{{ with $package.jurisdiction }}
+			<div class="card_flag flag flag-{{ . }}"><span>{{ . }}</span></div>
```

Comment:
Don't we have to apply the same slicing as in the packages page?
```suggestion
			<div class="card_flag flag flag-{{ slicestr . 0 2 }}"><span>{{ . }}</span></div>
```
If so, this might be a sign that this `<div>` would be better off in a dedicated component / partial that we can just call with the `jurisdiction` without thinking twice 🙂

Reply:
Good idea, I just implemented it with https://github.com/openfisca/openfisca.org/pull/134/commits/abdf930afc4f3c2d0956a41e4f810cbf369b98a0 👍